pkg/platform/abstract/platform.go

```diff
@@ -492,9 +510,14 @@ func (ap *Platform) prettifyProcessorLogLine(log []byte) (string, string, error)
 		logStruct.Time = &unparsedTime
 		logStruct.Level = wrapperLogStruct.Level
 		logStruct.Message = wrapperLogStruct.Message
+		logStruct.Name = wrapperLogStruct.Name
 
-		more := common.CreateKeyValuePairs(wrapperLogStruct.With)
-		logStruct.More = &more
+		if wrapperLogStruct.With != nil {
```

Comment:
Suggest to move the nil check inside `common.CreateKeyValuePairs` to keep this clean

Reply:
Doing this check to perform this line:
`workerID = wrapperLogStruct.With["worker_id"]` 
and CreateKeyValuePairs already supports _nil_ input